lib/aruba/platforms/windows_command_string.rb

```diff
@@ -11,13 +11,18 @@ module Platforms
     # @private
     class WindowsCommandString < SimpleDelegator
       def initialize(cmd)
-        __setobj__ format('%s /c "%s"', Aruba.platform.which('cmd.exe'), cmd)
+        __setobj__ cmd
```

Comment:
This is the default right? No explicit need to implement the constructor?

Reply:
You may be right. I'll take a look.

Reply:
Yes, it is no longer needed. I removed it.